ssdutil/ssdvendors/sandisk.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python

```

Comment:
Hi Cary,Also please delete this file.This is the test code not the official code.